app.rb

```diff
@@ -272,7 +272,7 @@ def state_uptimes_between(t1, t2)
         end
       end
       
-      if monitor_time
+      if monitor_time > 0
```

Comment:
`nil` is already handled in this algorithm up on [lines 247-255](https://github.com/codeforamerica/snap-it-up/pull/65/files#diff-cc95738088603531796e0d0f246a5d77R248).

The problem here was merely dividing by zero because a non-accepted incident covered the entire timeframe, which makes the total possible uptime (`monitor_time`) 0.